infra.bs

```diff
@@ -710,6 +714,10 @@ ordered sets; implementations can optimize based on the fact that the order is n
 <a for=list>contains</a> the given <a for=set>item</a>, or to perform the normal <a>list</a>
 <a for=list>prepend</a> operation otherwise.
 
+<p>To <dfn export for=set>replace</dfn> zero or more <a for=set>items</a> with a given
+<a for=set>item</a> is to replace the first item from the set that matches a given condition with
+the given item and remove the other items, or do nothing if none do.
```

Comment:
"other matching items"; "none match". This sentence is more complex than the "remove" counterpart so needs a bit more clarity.